src/action/interactive.rs

```diff
@@ -82,6 +82,9 @@ where
     pub suggestion: &'s Suggestion<'t>,
     /// The content the user provided for the suggestion, if any.
     pub custom_replacement: String,
+    pub i_custom_replacement_end_index: (u16, u16),
```

Comment:
I would prefer this to be without `i_` prefix and not a tuple, after all we only ever access `.1`

Reply:
Initially they has no this prefix but after I've added in with consideration to mark it as it's only used by `insert mode`.
yes we don't use `.0`

Reply:
If you look at my further comments, I am inclined to say there will be only one integer left as state - if there should remain more, then we should gather them in a separate struct as you said.